content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -434,6 +434,59 @@ The annotation `kubernetes.io/limit-ranger` records that resource defaults were
 and they were applied successfully.
 For more details, read about [LimitRanges](/docs/concepts/policy/limit-range).
 
+### kubernetes.io/config.hash
+
+Type: Annotation
+
+Example: `kubernetes.io/config.hash: "df7cc47f8477b6b1226d7d23a904867b"`
+
+Used on: Pod
+
+When the kubelet creates a static Pod based on a given manifest, it attaches this annotation
+to the static Pod. The value of the annotation is the UID of the Pod.
+Note that the kubelet also sets the `.spec.nodeName` to the current node name as if the Pod
+was scheduled to the node.
+
+### kubernetes.io/config.mirror
+
+Type: Annotation
+
+Example: `kubernetes.io/config.mirror: "df7cc47f8477b6b1226d7d23a904867b"`
+
+Used on: Pod
+
+For a static Pod created by the kubelet on a node, a {{< glossary_tooltip text="mirror Pod" term_id="mirror-pod" >}}
+is created on the API server. The kubelet adds an annotation to indicate that this Pod is
+actually a mirror Pod. The annotation value is copied from the [`kubernetes.io/config.hash`](#kubernetes-io-config-hash)
+annotation, which is usually the UID of the Pod.
+
+When updating a Pod with this annotation set, the annotation cannot be changed or removed.
+If a Pod doesn't have this annotation, it cannot be added during a Pod update.
+
+### kubernetes.io/config.source
+
+Type: Annotation
+
+Example: `kubernetes.io/config.source: "file"`
+
+Used on: Pod
+
+This annotation is added by the kubelet to indicate where the Pod comes from.
+For static Pods, the annoation value could be one of `"file"` or `"http"` depending
+on where the Pod manifest is located. For a Pod created on the API server and then
```

Comment:
nit: The possible values are: `file`, `http`, or `api`? Do you need to double quote the values and surround in code blocks?
nit: Spelling error: annoation -> annotation

Reply:
I was trying to emphasize that the value are strings. The double quotes can be removed.